src/editorIdentity.ts

```diff
@@ -1,7 +1,7 @@
 import * as vscode from 'vscode';
 
 /**
- * We consider two editors to be the same iff their EditorIdentities are the same
```

Comment:
"iff" here is actually shorthand for "if and only if", though it's probably worth writing out because otherwise it can look like a typo hahaha

Reply:
OK, I'll leave it here :-P